src/pages/admin/points.tsx

```diff
@@ -76,7 +76,7 @@ const AwardPointsPage: NextPage = () => {
 export default AwardPointsPage;
 
 const getServerSidePropsFunc: GetServerSideProps = async () => ({
-  props: {},
+  props: { title: 'Award Bonus Points', description: 'Grant Bonus Points to a Specific User' },
```

Comment:
nit: just to be consistent, maybe:
```suggestion
  props: { title: 'Award Bonus Points', description: 'Grant bonus points to a specific user' },
```